src/shell.js

```diff
@@ -117,6 +121,11 @@ var read_,
     readBinary,
     setWindowTitle;
 
+#if ENVIRONMENT_MAY_BE_NODE
```

Comment:
can we combine this with the #if ENVIRONMENT_MAY_BE_NODE) on line 129?

Reply:
Sure, fixing.